spec/models/claims_spec.rb

```diff
@@ -30,4 +30,52 @@
       expect(assignment.errors.of_kind?(:submission, :taken)).to be(true)
     end
   end
+
+  describe '#rejected?' do
+    it 'if rejected is true' do
+      claim.state = Nsm::MakeDecisionForm::REJECTED
+      expect(claim).to be_rejected
+    end
+  end
+
+  describe '#granted?' do
+    it 'if granted is true' do
+      claim.state = Nsm::MakeDecisionForm::GRANTED
+      expect(claim).to be_granted
+    end
+  end
+
+  describe '#formatted_allowed_total' do
+    context 'granted' do
+      it 'adjusted cost if adjusted more than claimed' do
+        claim = create(:claim, :increase_adjustment)
+        claim.state = Nsm::MakeDecisionForm::GRANTED
+        expect(claim).not_to receive(:formatted_claimed_total)
+        claim.formatted_allowed_total
```

Comment:
I think the tests should check that the output is what's expected here. E.g. at least that the allowed cost is greater than the claimed cost. And the next should check that the allowed cost is equal to the claimed cost. Otherwise this test just tests the implementation, not the behaviour.